### PR TITLE
Mpuncel/remove reality index uninstall

### PR DESCRIPTION
**Pull request by mpuncel**

### CODE REVIEW

**Comment by mpuncel** [comment]
LMK if you guys think the `current_manifest.yaml` change is stupid. That shouldn't be a state we encounter but i guess uninstall loops because of that file missing seem wrong

File: pkg/pods/pod.go [hunk 1]
```diff
@@ -352,30 +352,33 @@ func (pod *Pod) EnvDir() string {
...
 		return err
 	}
 
-	// halt launchables
-	for _, launchable := range launchables {
-		disableFunc := func() {
-			err = launchable.Disable()
-		}
-		pod.withTimeWarnings("disable", launchable.ServiceID(), disableFunc)
+	if err == nil {
```
Comment (anthonybishopric) [inline pkg/pods/pod.go]:
```
if err == NoCurrentManifest { return }
```

**Comment by anthonybishopric** [comment]
👍  to eliminating an endless looping behavior

File: pkg/pods/pod.go (hunk 1 above)
Comment (mpuncel) [inline pkg/pods/pod.go]: i'll move the inner stuff to a function so it's easier to read

**Comment by mpuncel** [comment]
might want to give this a once over again, the first commit is rewritten for clarity (but behavior is the same)

**Pull request merged**